include/ncengine/scene/Scene.h

```diff
@@ -17,8 +18,29 @@ class Registry;
 class Scene
 {
     public:
+        Scene(Vector3 extents = Vector3{150.0f, 150.0f, 150.0f})
+        : m_extents{Vector3{std::max(1.0f, extents.x), std::max(1.0f, extents.y), std::max(1.0f, extents.z)}}
```

Comment:
I don't know if we decision off the minimum value elsewhere, but if so, it would be nice to have a `g_minSceneExtent` constant.